content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst

```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
 Create a mandate
 ----------------
 
-The SEPA Direct Debit Mandate is the document that your customers sign to authorize you to collect
+The :abbr:`SDD` Mandate is the document that your customers sign to authorize you to collect
```

Comment:
Abbreviations must follow [this format](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#abbreviations) to work.